doc/source/tutorial/optimize.rst

```diff
@@ -1342,10 +1342,45 @@ residual is expensive to compute, good preconditioning can be crucial
 --- it can even decide whether the problem is solvable in practice or
 not.
 
+
 Preconditioning is an art, science, and industry. Here, we were lucky
 in making a simple choice that worked reasonably well, but there is a
 lot more depth to this topic than is shown here.
 
+
+Linear Sum Assignment
```

Comment:
This chapter should be in a section of its own; right now it's a subsection of "Root finding", but the two problems are not very related. This also means that the references end up looking like references for the linear sum assignment problem, when they're really not.

Reply:
Sure.